Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -243,4 +256,53 @@ def output_unbottled(formulae, deps_hash, noun, hash, any_named_args)
 
     puts "No unbottled dependencies found!"
   end
+
+  def output_lost_bottles
+    # $ git log --patch -G\^\ \+sha256.\*\ sonoma: --since=@\{\'1\ week\ ago\'\}
+    git_log = %w[git log --patch]
```

Comment:
```suggestion
    git_log = %w[git log --patch --no-ext-diff]
```

maybe? Not entirely clear what the behaviour of external diff tools is with `git log --patch` - but we don't want it in case it does do something.

Reply:
Very good idea!

Reply:
Handled in https://github.com/Homebrew/brew/pull/16115/commits/83598bb950bdb55b23232f7fe6767852aa856a23